notifications_utils/clients/zendesk/zendesk_client.py

```diff
@@ -0,0 +1,88 @@
+import requests
+from flask import current_app
+
+
+class ZendeskError(Exception):
+    def __init__(self, response):
+        self.response = response
+
+
+class ZendeskClient():
+    PRIORITY_URGENT = 'urgent'
+    PRIORITY_HIGH = 'high'
+    PRIORITY_NORMAL = 'normal'
+    PRIORITY_LOW = 'low'
+
+    TYPE_PROBLEM = 'problem'
+    TYPE_INCIDENT = 'incident'
+    TYPE_QUESTION = 'question'
+    TYPE_TASK = 'task'
+
+    TAGS_P2 = ['govuk_notify_support']
+    TAGS_P1 = ['govuk_notify_emergency']
+
+    # Group: 3rd Line--Notify Support
+    NOTIFY_GROUP_ID = 360000036529
+
+    # Organization: GDS
+    NOTIFY_ORG_ID = 21891972
+
+    def __init__(self):
+        self.api_key = None
+        self.api_host = None
+
+        self.department_id = None
+        self.agent_team_id = None
+        self.default_person_email = None
+
+    def init_app(self, app, *args, **kwargs):
+        self.api_key = app.config.get('ZENDESK_API_KEY')
+
+    def create_ticket(
+        self,
+        subject,
+        message,
+        ticket_type,
+        p1=False,
+        user_name=None,
+        user_email=None
+    ):
+        data = {
+            'ticket': {
+                'subject': subject,
+                'comment': {
+                    'body': message
+                },
+                'group_id': ZendeskClient.NOTIFY_GROUP_ID,
```

Comment:
`self.NOTIFY_GROUP_ID` is shorter (and better if we ever want to subclass this client).